src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionContextExtensions.cs

```diff
@@ -76,6 +84,94 @@ public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateEventArg
         }
 
         /// <summary>
+        /// The create mixed bash event args list.
+        /// </summary>
+        /// <param name="suspensionContext">The suspension context.</param>
+        /// <typeparam name="T">The type of collection item.</typeparam>
+        /// <returns>The <see cref="ICollection{NotifyRangedCollectionChangedEventArgs}"/>.</returns>
+        public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateMixedBashEventArgsList<T>(this SuspensionContext<T> suspensionContext)
+        {
+            Argument.IsNotNull(nameof(suspensionContext), suspensionContext);
+            Argument.IsValid(nameof(suspensionContext.Mode), suspensionContext.Mode, mode => mode == SuspensionMode.MixedBash);
+
+            return suspensionContext.CreateBashEventArgsList(SuspensionMode.MixedBash);
+        }
+
+        /// <summary>
+        /// The create mixed consolidate event args list.
+        /// </summary>
+        /// <param name="suspensionContext">The suspension context.</param>
+        /// <typeparam name="T">The type of collection item.</typeparam>
+        /// <returns>The <see cref="ICollection{NotifyRangedCollectionChangedEventArgs}"/>.</returns>
+        public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateMixedConsolidateEventArgsList<T>(this SuspensionContext<T> suspensionContext)
+        {
+            Argument.IsNotNull(nameof(suspensionContext), suspensionContext);
+            Argument.IsValid(nameof(suspensionContext.Mode), suspensionContext.Mode, mode => mode == SuspensionMode.MixedConsolidate);
+
+            var events = suspensionContext.CreateBashEventArgsList(SuspensionMode.MixedConsolidate);
```

Comment:
@alexfdezsauco forgot all other comments ... I have overseen this line. I am really sorry.